addons/sys_bf888s/radio/fnc_getSpatial.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Returns the spatial configuration of the radio.
+ *
+ * Arguments:
+ * 0: Radio ID <STRING> (Unused)
```

Comment:
Is it really unused ? It's reffed in the code below, so I suspect it's an accidental copypaste.

Reply:
```suggestion
 * 0: Radio ID <STRING>
```